apps/crn-frontend/src/news/api.ts

```diff
@@ -1,14 +1,17 @@
 import { ListNewsResponse, NewsResponse } from '@asap-hub/model';
-import { createListApiUrl, GetListOptions } from '../api-util';
+import { createListApiUrl, GetListOptions } from '@asap-hub/frontend-utils';
 import { API_BASE_URL } from '../config';
 
 export const getNews = async (
   options: GetListOptions,
   authorization: string,
 ): Promise<ListNewsResponse> => {
-  const resp = await fetch(createListApiUrl('news', options).toString(), {
-    headers: { authorization },
-  });
+  const resp = await fetch(
+    createListApiUrl(new URL('news', `${API_BASE_URL}/`), options).toString(),
```

Comment:
Maybe each project needs a local getApiBaseURL. I don't like new URL xxx all the time. Its begging for the trailing slash to be ommited